client.go

```diff
@@ -186,7 +168,7 @@ func (c *Client) RequestToken(ctx context.Context) error {
 
 	buf := bytes.NewBuffer([]byte("grant_type=client_credentials"))
 
-	req, err := http.NewRequest("POST", c.TokenURL(), buf)
+	req, err := http.NewRequest("POST", c.BuildAPIURL("token"), buf)
```

Comment:
To be clear, I'm find with `TokenURL` no longer being used by the client. Just don't want to remove the method without a major version bump to preserve API continuity.

Reply:
makes sense! changes incoming